release_notes/Service_Fabric_ReleaseNotes_90CU9.md

```diff
@@ -0,0 +1,61 @@
+|.NET SDK |Windows .NET SDK <br> Microsoft.ServiceFabric <br> Reliable Services and Reliable Actors <br> ASP.NET Core Service Fabric integration| 6.0.1526  <br> 9.0.1526 <br> 6.0.1526 <br> 6.0.1526 |
+|Java SDK  |Java for Linux SDK  | 1.0.6 |
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 11.0.1 |
+
+## Service Fabric Feature and Bug Fixes
+| Versions | IssueType | Description | Resolution | 
+|-|-|-|-|
+| **Windows - 9.0.1526.9590<br>Ubuntu 18 - 9.0.1463.1<br>Ubuntu 20 - 9.0.1463.1** | **Feature** | Key Value Store (KVS) | **Brief Description**:  KVS backed partitions, including Actors and System Services, experience quorum loss due to StoreLVIDLimitHit error code (0x80071d4d), indicating the depletion of values for a monotonically increasing ID field. This results in unavailability of the partition until a manual offline maintenance task is executed to resolve the issue. <br>**Solution**: To address the error, KVS replicas now restart automatically and initiate the maintenance task (Extensible Storage Engine database compaction) during the next open. While there may be temporary availability loss in partitions depending on the task duration, no manual intervention is required to mitigate the issue.
+| **Windows - 9.0.1526.9590<br>Ubuntu 18 - 9.0.1463.1<br>Ubuntu 20 - 9.0.1463.1** | **Feature** | Key Value Store (KVS) | **Brief Description**: Auto-compaction now emits health events at the start and completion, visible in the Service Fabric Explorer, enhancing its monitoring capabilities. These events complement trace data, providing effective visibility into active auto-compaction progress.
+
+## Retirement and Deprecation Path Callouts
+* As aligned with [Microsoft .NET and .NET Core - Microsoft Lifecycle | Microsoft Learn](https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-and-net-core), SF Runtime has dropped support for Net Core 3.1 as of December 2022. For  supported versions see [Service Fabric supported Linux versions](https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-versions#current-versions-1) and migrate applications as needed.
+* Ubuntu 18.04 LTS will reach its 5-year end-of-life window on June, 2023. Service Fabric runtime will drop support for 18.04 LTS after the published date, and we recommend moving your clusters and applications to supported versions listed here: [Service Fabric supported Linux versions](https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-versions#supported-linux-versions-and-support-end-date)
+* Service Fabric runtime will soon stop supporting BinaryFormatter based remoting exception serialization by default and move to using Data Contract Serialization (DCS) based remoting exception serialization by default. Current applications using it will continue to work as-is, but Service Fabric strongly recommends moving to using Data Contract Serialization (DCS) based remoting exception instead.
```

Comment:
Service Fabric runtime will soon stop supporting BinaryFormatter

Are we actually dropping support for BinaryFormatter if customers use it or just modifying the default?

Reply:
We will be dropping the support for BinaryFormatter in future.